unobot.py

```diff
@@ -462,14 +463,14 @@ def get_card(self):
     @staticmethod
     def create_deck():
         new_deck = []
-        for card in COLORED_CARD_NUMS:
+        for card in (COLORED_CARD_NUMS + COLORED_CARD_NUMS[1:]):
```

Comment:
**Code analysis:**
4(0-9 + R + S + D2 + 1-9 + R + S + D2) + 2(W) + 2(WD4) =
4(10 + 1 + 1 + 1 + 9 + 1 + 1 + 1) + 2 + 2 =
4(25) + 4 = **104**†

Not even gonna bother with debug output yet, because that's wrong. There aren't enough W/WD4 cards.

† Actually **208** because `new_deck *= 2` before `return new_deck`, but that's not how many cards are actually _generated_.